engine/schema/src/main/resources/META-INF/db/schema-41710to41800.sql

```diff
@@ -37,3 +37,128 @@ ALTER TABLE `cloud`.`vpc`
     ADD COLUMN `dns2` varchar(255) DEFAULT NULL COMMENT 'second IPv4 DNS for the vpc' AFTER `dns1`,
     ADD COLUMN `ip6dns1` varchar(255) DEFAULT NULL COMMENT 'first IPv6 DNS for the vpc' AFTER `dns2`,
     ADD COLUMN `ip6dns2` varchar(255) DEFAULT NULL COMMENT 'second IPv6 DNS for the vpc' AFTER `ip6dns1`;
+
+ALTER TABLE `cloud`.`network_offerings` ADD COLUMN `for_tungsten` int(1) unsigned DEFAULT '0' COMMENT 'is tungsten enabled for the resource';
```

Comment:
I would advise to use the stored procedure `IDEM_POTENT_ADD_COLUMN()`. It is being defined in several upgrade files. As I know you have been developing this since at least 4.14, you will hit upgrade problems going to 4.18 or above if you don´t do this at this point.

Reply:
i updated db script as your suggestion